recipes-kernel/linux/linux-raspberrypi_5.4.bb

```diff
@@ -11,4 +11,5 @@ SRC_URI += "file://0001-Revert-selftests-bpf-Skip-perf-hw-events-test-if-the.pat
             file://0002-Revert-selftests-bpf-Fix-perf_buffer-test-on-systems.patch \
             file://powersave.cfg \
             file://android-drivers.cfg \
+	    file://my_config.cfg \
```

Comment:
The file name is not appropriate perhaps it could be a tft.conf or add specific tft name to qualify it more

Reply:
I suspect this is for piscreen.